### PR TITLE
Update ad-hoc-distributed-queries-server-configuration-option.md

### PR DESCRIPTION
**Pull request by JMNetwalker**
Azure SQL Database and Azure SQL Managed Instance for reference

### CODE REVIEW

**Comment by PRMerger16** [comment]
@JMNetwalker : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/database-engine/configure-windows/ad-hoc-distributed-queries-server-configuration-option.md [hunk 1]
```diff
@@ -52,7 +52,9 @@ FROM OPENROWSET('SQLNCLI', 'Server=Seattle1;Trusted_Connection=yes;',
       ORDER BY GroupName, Name') AS a;  
 GO  
 ```  
-  
+## Azure SQL Database and Azure SQL Managed Instance
+See this [Features comparison: Azure SQL Database and Azure SQL Managed Instance](https://docs.microsoft.com/en-us/azure/azure-sql/database/features-comparison) for reference
```
Comment (ShannonLeavitt) [inline docs/database-engine/configure-windows/ad-hoc-distributed-queries-server-configuration-option.md:56]:
```suggestion
See the [Features comparison: Azure SQL Database and Azure SQL Managed Instance](https://docs.microsoft.com/azure/azure-sql/database/features-comparison) for reference. 

```
Removing locale identifier and adding period

**Comment by PRMerger9** [comment]
@JMNetwalker : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by markingmyname — APPROVED** [review]
#sign-off

**Comment by markingmyname** [comment]
#sign-off

**Pull request merged**